docs/framework/20categories/20architecture/liquidity-networks.md

```diff
@@ -1,3 +1,24 @@
 ### Liquidity Networks
+Liquidity networks enable users to transfer already minted assets from one chain to another. With such protocols, there's no ‘bridging’ of assets, i.e., no assets move across chains. Instead, assets are exchanged between one user (liquidity providers, aka LPs) and another bridge user on different networks, resulting in a cross-chain swap. For example, a user receives the desired asset on the destination network via the LP. In return, the LP receives the user’s assets on the source network and fees for providing the desired asset. 
 
-Note: include HTLCs as a possible design approach
\ No newline at end of file
+Typically, LP funds within a liquidity network are collected and locked in a smart contract (liquidity pool) to facilitate quick cross-chain swaps. When a user initiates a transaction, the desired assets are released from the smart contract on the destination network. In essence, liquidity networks work similarly to peer-to-peer networks where LPs, acting as routers, own and distribute liquidity across chains. 
+
+The relationship between the LP and the users depends on the mechanism used by the liquidity network:
+
+**Atomic swaps using HTLCs (locally verified)** 
```

Comment:
Should we discuss the off-chain mechanics that enable discovery (i.e., user finding an LP), auctioning, etc., and any concerns that might emerge from that?

Reply:
Not sure if we need to go into detail about the off-chain mechanics but there can be considerations such as: 1) is the network of LPs whitelisted or can anyone become an LP. 2) If there is a slashing mechanism to penalise LPs that accept bids but do not fulfil them and thus can grief users if they wish. What do you think?

Reply:
Right. I mention the first point in the [updated considerations section from PR #43](https://crosschainriskframework.github.io/framework/20categories/20architecture/architecture/#liquidity-networks). 

I hadn't thought of point 2, which I think is a good one to add.

Reply:
Considerations to add:

1) Does the user always receive native assets or can they receive canonical tokens minted by the underlying messaging protocols?
2) Can LPs grief users by rejecting to fulfil exchange requests? If so, is there a slashing mechanism to penalize such behaviour?

Reply:
Agreed with both. The second one is somewhat mentioned through the point on fairness and optionality. But the way it is written is probably a bit abstract, and it does not discuss mitigations like slashing. I can update it.

Reply:
Addressed in PR #60